frontend/src/metabase/nav/components/DatabasePromptBanner/DatabasePromptBanner.tsx

```diff
@@ -0,0 +1,59 @@
+import { t } from "ttag";
```

Comment:
@ranquild I added one test that test a happy path of this feature, the rest should already be covered with the integration test.